src/drift.core/model/Namespace.cs

```diff
@@ -17,5 +19,15 @@ public class Namespace
         public string[] Deny { get; set; }
 
         public string Name { get; set; }
+
+        public Namespace Expand(TeamProject teamProject)
+        {
+            var result = new Namespace();
```

Comment:
Can be simplified to:
            return new Namespace
            {
                Allow = this.Allow.Select(a => a.Expand(teamProject)).ToArray(),
                Deny = this.Deny.Select(d => d.Expand(teamProject)).ToArray(),
                Name = this.Name.Expand(teamProject),
            };